install_gcloud.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+#Install gcloud
+if [ ! -d $HOME/gcloud/google-cloud-sdk ]; then
```

Comment:
@wbrockman Could you give us a :+1: on using these several lines of google code?